README.md

```diff
@@ -21,7 +21,9 @@ Secondary features :
 !ctf addchallenge <challenge_name>
 !ctf working [challenge_name]
 !ctf status
-!ctf solved [challenge_name] [support_member]
+!ctf solve [challenge_name] [support_member]
```

Comment:
:+1:

Reply:
Your new lines below use the right format (`<>`). Let's change this one to the same format (`[]` is usually for _optional_ fields)